index.bs

```diff
@@ -13397,7 +13397,7 @@ Each {{DOMException}} object has an associated <dfn for="DOMException">name</dfn
 The <dfn constructor for="DOMException"><code>DOMException(|message|, |name|)</code></dfn>
 constructor, when invoked, must run these steps:
 
-1. Set <b>this</b>'s [=DOMException/name=] to |name|.
+1. Set <b>this</b>'s [=DOMExcep tion/name=] to |name|.
```

Comment:
Maybe this space is added accidentally 👀?

Reply:
Thanks! Fixed.